src/nvim/statusline.c

```diff
@@ -929,7 +929,7 @@ int build_stl_str_hl(win_T *wp, char *out, size_t outlen, char *fmt, int use_san
           group_len = (min_group_width - group_len) * utf_char2len(fillchar);
           memmove(t + group_len, t, (size_t)(out_p - t));
           if (out_p + group_len >= (out_end_p + 1)) {
-            group_len = (long)(out_end_p - out_p);
+            group_len = (long)(out_end_p - out_p);  // NOLINT(google-readability-casting)
```

Comment:
It's also possible to change `group_len` and `min_group_width` to `ptrdiff_t` and remove the cast here.